streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBTimeOrderedWindowStore.java

```diff
@@ -33,172 +39,112 @@
  * For key range queries, like fetch(key, fromTime, toTime), use the {@link RocksDBWindowStore}
  * which uses the {@link WindowKeySchema} to serialize the record bytes for efficient key queries.
  */
+@SuppressWarnings("unchecked")
 public class RocksDBTimeOrderedWindowStore
```

Comment:
Two things:
- Seems `TimeOrderedKeySchema` is not needed anymore. Should the class be removed?
- Should we rename the class to remove the `Window` par?

Reply:
Sounds good, will do.